src/lib/y2storage/setup_checker.rb

```diff
@@ -105,8 +107,51 @@ def mount_warnings
       devicegraph.mount_points.map { |mp| mount_warning(mp) }.compact
     end
 
+    # Warnings detected in the setup for each enabled policy
+    #
+    # @return [Hash<Y2Security::SecurityPolicies::Policy, Array<SetupError>>]
+    def security_policies_warnings
+      return {} unless Yast::Mode.installation
+
+      return @security_policies_warnings if @security_policies_warnings
+
+      policies_warnings = security_policies_issues.map do |policy, issues|
+        warnings = issues.map { |i| SetupError.new(message: i.message) }
+        [policy, warnings]
+      end
+
+      @security_policies_warnings = policies_warnings.to_h
+    end
+
     private
 
+    # Issues for each enabled security policy
+    #
+    # @note yast2-security might not be available, see {#ensure_security_policies}.
+    #
+    # @return [Hash<Y2Security::SecurityPolicies::Policy, Array<Y2Security::SecurityPolicies::Issue>>]
+    def security_policies_issues
+      issues = ensure_security_policies do
+        policies_manager = Y2Security::SecurityPolicies::Manager.instance
+        scope = Y2Security::SecurityPolicies::Scopes::Storage.new(devicegraph: devicegraph)
+
+        policies_manager.issues(scope).to_h
+      end
+
+      issues || {}
+    end
+
+    # Ensures security polices are correctly loaded
+    #
+    # Package yast2-security is not added as dependency to avoid cyclic dependencies.
```

Comment:
Please list the known use cases, and whether yast2-security is present/absent
Like
1. Installation time: yes, because yast2-security is included
2. RPM build time: no, avoiding cyclic build dependency
3. ... ?